platform/authorized_api/apps/index.json

```diff
@@ -289,34 +294,37 @@
         },
         "delay": {
           "type": "integer",
-          "description": "Delay for an app",
+          "description": "Delay (in ms) is used to slow down actions (e.g. redirect) when waiting for tags to be resolved",
```

Comment:
Maybe something like this:

Delay added before some tracked actions. It's necessary to track actions that can change URL of displayed page (e.g. click on a new page link or download). Delay is provided in milliseconds.

I'm not sure what "when waiting for tags to be resolved" means.
Default value may also be helpful (if it's 500 - we can skip example instead).